src/registry_schemas/schemas/restoration.json

```diff
@@ -21,16 +24,67 @@
                 },
                 "type": {
                     "type": "string",
-                    "title": "The Type of Restoration, such as Full Restoration, Limited Restoration, etc.",
+                    "title": "The Type of Restoration, such as Full Restoration, Limited Restoration, Limited Restoration Extension, Limited Restoration to Full.",
                     "enum": [
                         "fullRestoration",
-                        "limitedRestoration"
+                        "limitedRestoration",
+                        "limitedRestorationExtension",
+                        "limitedRestorationToFull"
```

Comment:
I like the idea of `limitedRestoration` -> `fullRestoration` instead of `limitedRestorationToFull` but we have separate fee code and validation. We might need to do some additional work to identify the type of filing. @argush3 do you foresee any issue if we do that

Reply:
FWIF, here's how the filing codes are labelled in the UI:
```
  RESTORATION_FULL = 'RESTF', // for BC/BEN/CC/ULC
  RESTORATION_LIMITED = 'RESTL', // for BC/BEN/CC/ULC
  RESTORATION_LTD_TO_FULL = 'RESXF', // for BC/BEN/CC/ULC
  RESTORATION_LTD_EXTEND = 'RESXL', // for BC/BEN/CC/ULC
```

Reply:
yeah we would need to figure out the fee code on the BE by just going from limited->full.  But I can't think of any major issue doing that off the top of my head.

I think it's more straightforward to just have a separate sub-type for `limitedRestorationToFull` though.

Reply:
I am thinking the same. @thorwolpert are you ok to go with this now?

Reply:
IMHO, we should validate that a "limited to full" filing is only valid while the business is in limited resto state, but I too prefer the "stateless" approach of including the subtype in the filing JSON.

Another + for this is that you could tell what's going on when looking up filings manually in the db.